deps/oob-1.0.3/sf_ocsp_telemetry_data.c

```diff
@@ -0,0 +1,134 @@
+/*
+* Copyright (c) 2018-2020 Snowflake Computing
+*/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "sf_ocsp_telemetry_data.h"
+
+SF_OTD *get_ocsp_telemetry_instance()
+{
+  return (SF_OTD *)calloc(1, sizeof(SF_OTD));
+}
+
+void sf_otd_set_event_type(char *event_type, SF_OTD* ocsp_telemetry_data)
+{
+  if (!ocsp_telemetry_data)
+  {
+    return;
+  }
+
+  snprintf(ocsp_telemetry_data->event_type,OCSP_TELEMETRY_EVENT_MAX_LEN, event_type);
+}
+
```

Comment:
Why aren't all of the functions in this file inline functions?

Reply:
They could have been inline, I thought I'd just create a new placeholder if we want to add more functionality later. Wouldn't want to pollute the header file. It wouldn't harm much to put them inline but that's what the thought process was behind the decision.